generator/generator_tool/generator_tool.cpp

```diff
@@ -73,6 +73,9 @@ DEFINE_bool(make_routing, false, "Make routing info based on osrm file.");
 DEFINE_bool(make_cross_section, false, "Make cross section in routing file for cross mwm routing (for OSRM routing).");
 DEFINE_bool(make_routing_index, false, "Make sections with the routing information.");
 DEFINE_bool(make_cross_mwm, false, "Make section for cross mwm routing (for dynamic indexed routing).");
+DEFINE_uint64(
+    cross_mwm_progress_peroid, 10,
+    "The smaller the period, the more often progress logs are printed. Set 0 to disable logs.");
```

Comment:
So, what's the scale? I mean, what's the difference between 10 and, say, 11? Or between 10 and 10000?